style.css

```diff
@@ -24,10 +33,25 @@ section {
   border-radius: 10%;
   margin-top: 100px;
   margin-bottom: 100px;
+  margin-right: 100px;
+  margin-left: 100px;
+  padding: 40px;
+  font-size: 25px;
+}
+
+ul {
+  background-color: white;
+  background-size: 200%;
+  border-radius: 10%;
+  text-align: left;
+  background-color: white;
+  border-radius: 10%;
+  margin-top: 100px;
+  margin-bottom: 100px;
   margin-right: 150px;
   margin-left: 80px;
```

Comment:
Instead of assigning a margin to each side on separate lines, you can shorthand the margin property with Top Right Bottom Left (or TRBL -- trouble to remember it), e.g. margin: 100px 150px 100px 80px;